pegasus/data/csv.rb

```diff
@@ -0,0 +1,199 @@
+require 'cdo/chat_client'
```

Comment:
This file was extracted from `seed.rake` so that the class could be reused outside of the Rake task, it's otherwise unchanged.